### PR TITLE
chore: PEx v2 input_descriptors schema/type requirement example update 

### PR DESCRIPTION
**Pull request by rolsonquadras**
schema/type requirement example update based on DIF PEx v2

Reference - https://identity.foundation/presentation-exchange/#input-descriptor-object

### CODE REVIEW

File: spec/spec.md [hunk 1]
```diff
@@ -1171,7 +1175,11 @@ this property MUST be a string as described in
               {
                 "id": "vaccination_input",
                 "name": "Vaccination Certificate",
-                "schema": "https://w3id.org/vaccination/#VaccinationCertificate",
+                "@schema": [
```
Comment (OR13) [inline spec/spec.md:1178]: is this supposed to be `@schema` ?

File: spec/spec.md [hunk 2]
```diff
@@ -657,7 +657,11 @@ In the following message structure, the issuer wants a Permanent Resident Card (
                         "id":"prc_input",
                         "name":"Permanent Resident Card",
                         "purpose":"We need PRC to verify your status.",
-                        "schema":"https://w3id.org/citizenship#PermanentResidentCard",
+                        "schema":[
```
Comment (OR13) [inline spec/spec.md:660]: @rolsonquadras do this look right to you?

**Review by OR13 — APPROVED** [review]
I think this is correct, but pinging other folks for a review, since we changed this recently IIRC.

File: spec/spec.md (hunk 1 above)
Comment (rolsonquadras) [reply spec/spec.md]: my bad, fixed it now.

**Review by OR13 — CHANGES_REQUESTED** [review]
Suggest we remove the `schema` property, as its no longer required / supported in PE v2.

**Comment by bumblefudge** [comment]
https://identity.foundation/presentation-exchange/#input-descriptor-object
^ The current draft that migrates `schema` to `fields`. See discussion on https://github.com/decentralized-identity/presentation-exchange/issues/256

**Comment by rolsonquadras** [comment]
Thanks @OR13 @bumblefudge - Removed custom schema and added it as part of constraints (according to PEx v2)

**Review by OR13 — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Comment by rolsonquadras** [comment]
@brentzundel @OR13 - fixed merge conflicts

**Comment by brentzundel** [comment]
multiple positive reviews, feedback incorporated, merging

**Pull request merged**